data/denominations/amenity/place_of_worship.json

```diff
@@ -0,0 +1,52 @@
+{
+  "properties": {
+    "path": "denominations/amenity/place_of_worship",
```

Comment:
Besides places of worship, schools, community centers, and social facilities are often identified by their denomination as well. The editor would have to make sure not to “upgrade” something to a place of worship just because it has a matching denomination.

Reply:
That's a good point - I was thinking about adding schools to the denominations tree but left it out for now. IIRC iD only matches items of a different category if it falls into one of these [matchGroups](https://github.com/osmlab/name-suggestion-index/blob/main/dist/matchGroups.json)

Reply:
Yeah I think in this situation I think it should only offer to upgrade `amenity/place_of_worship`, or maybe also some generically tagged features like `building=yes` or `amenity=yes` when paired with a matching `name` or `operator`.